packages/nitrogen/src/views/CppHybridViewComponent.ts

```diff
@@ -86,14 +104,7 @@ ${createFileMetadataString(`${component}.hpp`)}
 
 #pragma once
 
-#include <optional>
-#include <NitroModules/NitroDefines.hpp>
-#include <NitroModules/NitroHash.hpp>
-#include <NitroModules/CachedProp.hpp>
-#include <react/renderer/core/ConcreteComponentDescriptor.h>
-#include <react/renderer/core/PropsParserContext.h>
-#include <react/renderer/components/view/ConcreteViewShadowNode.h>
-#include <react/renderer/components/view/ViewProps.h>
+${Array.from(fixedHeaderIncludes).join('\n')}
```

Comment:
This is again an `Array.from` - I think we copy arrays/sets 4 times here. Not necessary a problem with performance, but code is not clean.